src/main/java/io/bf2/kafka/common/PartitionCounter.java

```diff
@@ -157,7 +157,7 @@ public int getMaxPartitions() {
 
     /**
      * @return the value of the {@link Config#PRIVATE_TOPIC_PREFIX} key in the broker configs, or a default of
-     *         {@link Config#DEFAULT_PRIVATE_TOPIC_PREFIX} if not set.
+     *         {@link Config#DEFAULT_NO_PRIVATE_TOPIC_PREFIX} if not set.
```

Comment:
Is this rename intended? I'm not convinced that the change in name is particularly helpful.